### PR TITLE
[Referent Space] UI update integration

### PR DESCRIPTION
**Pull request by enzozccr**

### CODE REVIEW

File: front/style/layout/_buttons.scss [hunk 1]
```diff
@@ -191,6 +191,17 @@
     }
 }
 
+@mixin btn--ghosting--blue {
```
Comment (marineb) [inline front/style/layout/_buttons.scss]: What does `ghosting` stand for? Is it a naming that people will understand?

File: front/style/pages/_referent.scss [hunk 2]
```diff
@@ -1,4 +1,184 @@
...
+                    margin: 0;
+                    span {
+                        color: $black;
+                        font-weight: bold;
+                    }
+                }
+            }
+
+            .report {
+                // margin: 25px 0 0 0;
```
Comment (marineb) [inline front/style/pages/_referent.scss]: Remove comment if unnecessary

File: front/style/pages/_referent.scss [hunk 3]
```diff
@@ -1,4 +1,184 @@
...
+        margin-bottom: 68px;
+
+        .first-section {
+            display: flex;
+            flex-wrap: wrap;
+            flex-direction: column-reverse;
+            justify-content: space-between;
+            padding: 35px 0;
+
+            .el {
```
Comment (marineb) [inline front/style/pages/_referent.scss]: Isn't there a more specific class that can be used?

File: front/style/pages/_referent.scss [hunk 4]
```diff
@@ -103,6 +283,7 @@
         }
     }
 
+    // old (keep temporary)
```
Comment (marineb) [inline front/style/pages/_referent.scss]: Explain why we need to keep this, just add more details in the comment.

File: front/style/pages/_referent.scss [hunk 5]
```diff
@@ -126,56 +307,150 @@
         }
     }
 
-    & nav.referent-menu,
-    & div.referent-sub-menu {
-        text-align: center;
-        margin-bottom: 20px;
+    // updated
```
Comment (marineb) [inline front/style/pages/_referent.scss]: Why `updated`?

File: front/style/pages/_referent.scss [hunk 6]
```diff
@@ -126,56 +307,150 @@
...
+        flex-direction: row;
+        flex-wrap: wrap;
+        justify-content: space-between;
+        margin-bottom: 40px;
 
-            li {
-                display: inline-block;
-                margin: 0 20px;
+        h3 {
+            font-size: 24px;
```
Comment (marineb) [inline front/style/pages/_referent.scss]: Can't we use existing mixins for typography (font-size and line-height)?

File: front/style/pages/_referent.scss [hunk 7]
```diff
@@ -126,56 +307,150 @@
...
+            &:hover {
+                background-color: $blue--dark;
+                border: 1px solid $blue--dark;
+                color: white;
+            }
+        }
+
+        .datagrid__count {
+            color: $dark;
+            font-size: 16px;
```
Comment (marineb) [inline front/style/pages/_referent.scss]: Same comment on the font. Can't we use an existing mixin ?

File: front/style/pages/_referent.scss [hunk 8]
```diff
@@ -126,56 +307,150 @@
...
+
+        }
+    }
+
+    .datagrid__table-manager {
+        width: 100%;
+        text-align: left;
+        font-family: $font-roboto;
+        color: $dark;
+        font-size: 14px;
```
Comment (marineb) [inline front/style/pages/_referent.scss]: Mixin?

File: front/style/pages/_referent.scss [hunk 9]
```diff
@@ -126,56 +307,150 @@
...
+            }
+        }
+
+        @media (min-width: $mobile) {
+
+            justify-content: flex-start;
+
+            .datagrid__count {
+                margin-left: auto;
+                font-size: 18px;
```
Comment (marineb) [inline front/style/pages/_referent.scss]: Mixin?

File: front/style/pages/_referent.scss [hunk 10]
```diff
@@ -126,56 +307,150 @@
...
+        text-align: left;
+        font-family: $font-roboto;
+        color: $dark;
+        font-size: 14px;
+        margin-bottom: 30px;
+
+        thead {
+            background-color: #f6f9fc;
+            color: #7b889b;
+            font-size: 13px;
```
Comment (marineb) [inline front/style/pages/_referent.scss]: Mixin

File: templates/referent/_layout.html.twig [hunk 11]
```diff
@@ -14,61 +14,96 @@
...
-            </header>
+            <header class="manager-header">
+                <div class="l__wrapper">
+                    <div class="first-section">
+                        <div class="el">
+                            <p>Vous gérez : <span>{{ app.user.managedArea.tags|join(', ') }}</span></p>
+                            <p>Dernière connexion : <span>01/02/04</span></p>
+                        </div>
+                        <p class="report">🐛 Bug ? Nouveau besoin ?
+                            <a href="https://en-marche.helprace.com/" target="_blank" class="text--blue--dark link--no-decor">Écrivez-nous !</a>
```
Comment (marineb) [inline templates/referent/_layout.html.twig]: I think @guillaumedenhez wanted to link directly to the telegram channel.

File: front/style/pages/_referent.scss [hunk 12]
```diff
@@ -1,4 +1,184 @@
...
+                }
+            }
+        }
+
+        @media (min-width: $mobile) {
+
+            .first-section {
+                flex-direction: row;
+
+                .el {
```
Comment (marineb) [inline front/style/pages/_referent.scss]: More specific class than `el`?

File: front/style/pages/_referent.scss [hunk 13]
```diff
@@ -1,4 +1,184 @@
...
+                    }
+
+                    &:last-child {
+                        margin-right: 0;
+                    }
+
+                    & a {
+                        display: inline-block;
+                        padding: 5px 0 14px 0;
+                        color: #7b889b;
```
Comment (marineb) [inline front/style/pages/_referent.scss]: Can this color be a variable?

File: front/style/pages/_referent.scss [hunk 14]
```diff
@@ -1,4 +1,184 @@
...
+        &__menu {
+            font-family: $font-roboto;
+            font-weight: normal;
+            font-size: 17px;
+
+            ul {
+                display: flex;
+                flex-direction: row;
+                flex-wrap: wrap;
+                border-bottom: 1px solid #e5e9ee;
```
Comment (marineb) [inline front/style/pages/_referent.scss]: Can this color be a variable?

File: front/style/pages/_referent.scss [hunk 15]
```diff
@@ -1,4 +1,184 @@
...
+                .btn:first-child {
+                    margin-right: 10px;
+                }
+            }
+        }
+
+        &__menu {
+            font-family: $font-roboto;
+            font-weight: normal;
+            font-size: 17px;
```
Comment (marineb) [inline front/style/pages/_referent.scss]: Can this be a font mixin?

File: front/style/pages/_referent.scss [hunk 16]
```diff
@@ -1,4 +1,184 @@
...
+            flex-direction: column-reverse;
+            justify-content: space-between;
+            padding: 35px 0;
+
+            .el {
+                margin-top: 20px;
+                p {
+                    font-family: $font-roboto;
+                    font-size: 14px;
+                    color: #7b889b;
```
Comment (marineb) [inline front/style/pages/_referent.scss]: Can this color be a variable?

File: front/style/pages/_referent.scss [hunk 17]
```diff
@@ -1,4 +1,184 @@
 .referent {
+
+    .manager-header {
+        background-color: #f6f9fc;
```
Comment (marineb) [inline front/style/pages/_referent.scss]: Can this color be a variable?

File: front/style/layout/_buttons.scss (hunk 1 above)
Comment (enzozccr) [reply front/style/layout/_buttons.scss]: this is for buttons with transparent background. today only default one (black) are existing and they use the default clzass `.btn`, i've added the blue one.

File: front/style/pages/_referent.scss (hunk 17 above)
Comment (enzozccr) [reply front/style/pages/_referent.scss]: yes i still need to add the new variables colors, its wip

File: front/style/pages/_referent.scss (hunk 6 above)
Comment (enzozccr) [reply front/style/pages/_referent.scss]: well all existing ones are not fitting with new designs

File: front/style/layout/_layout.scss [hunk 18]
```diff
@@ -335,6 +335,10 @@
             margin-top: 40px;
         }
 
+        &--top-30 {
```
Comment (marineb) [inline front/style/layout/_layout.scss]: `top-30` and `top-large` have the same values. We should be using one, and if it's the `top-30`, then replace all the `top-large` in the code, and delete the class ! :)

File: front/style/manager-space/_base-layout.scss [hunk 19]
```diff
@@ -0,0 +1,185 @@
...
+                &.active a {
+                    color: $black;
+                    font-weight: bold;
+                    border-bottom-color: #46d5c7;
+                }
+            }
+        }
+    }
+
+    // I know, it's a custom size... but goes great like this
```
Comment (marineb) [inline front/style/manager-space/_base-layout.scss]: 😊

File: front/style/manager-space/_base-layout.scss [hunk 20]
```diff
@@ -0,0 +1,185 @@
...
+
+.tips-information {
+    max-width: 900px;
+    background-color: #fefbf0;
+    font-size: 15px;
+    font-weight: normal;
+    line-height: 20px;
+    padding: 20px;
+    margin-bottom: 60px;
+    color: #9a895f;
```
Comment (marineb) [inline front/style/manager-space/_base-layout.scss]: Color should be a variable :)

File: front/style/manager-space/_base-layout.scss [hunk 21]
```diff
@@ -0,0 +1,185 @@
...
+                }
+            }
+        }
+    }
+
+}
+
+.tips-information {
+    max-width: 900px;
+    background-color: #fefbf0;
```
Comment (marineb) [inline front/style/manager-space/_base-layout.scss]: Color should be a variable

File: front/style/manager-space/_filters.scss [hunk 22]
```diff
@@ -0,0 +1,188 @@
+@mixin filter-label {
+    display: block;
+    font-family: $font-roboto;
+    font-size: 13px;
```
Comment (marineb) [inline front/style/manager-space/_filters.scss]: Why not use a text mixin ? (add a new one if necessary)

File: front/style/manager-space/_filters.scss [hunk 23]
```diff
@@ -0,0 +1,188 @@
...
+    margin-bottom: 7px;
+    text-transform: uppercase;
+}
+
+@mixin filter-field {
+    box-sizing: border-box;
+    display: block;
+    width: 100%;
+    font-family: $font-roboto;
+    font-size: 14px;
```
Comment (marineb) [inline front/style/manager-space/_filters.scss]: Same comment re : text mixin.

File: front/style/manager-space/_filters.scss [hunk 24]
```diff
@@ -0,0 +1,188 @@
...
+    border: solid 1px #c9d4e4;
+    background-color: $white;
+}
+
+@mixin filter-radio-label {
+    box-sizing: border-box;
+    display: block;
+    width: 100%;
+    font-family: $font-roboto;
+    font-size: 14px;
```
Comment (marineb) [inline front/style/manager-space/_filters.scss]: Text mixin

File: front/style/manager-space/_filters.scss [hunk 25]
```diff
@@ -0,0 +1,188 @@
...
+    font-weight: normal;
+    line-height: normal;
+    color: $dark;
+    margin: 0;
+    height: 40px;
+    border: solid #c9d4e4;
```
Comment (marineb) [inline front/style/manager-space/_filters.scss]: Color variable

File: front/style/manager-space/_forms.scss [hunk 26]
```diff
@@ -0,0 +1,122 @@
+@mixin form-label {
+    display: block;
+    font-family: $font-roboto;
```
Comment (marineb) [inline front/style/manager-space/_forms.scss]: Text mixin

File: front/style/manager-space/_forms.scss [hunk 27]
```diff
@@ -0,0 +1,122 @@
...
+    font-weight: bold;
+    line-height: normal;
+    color: $black;
+    margin-bottom: 10px;
+}
+
+@mixin form-field {
+    display: block;
+    font-family: $font-roboto;
+    font-size: 16px;
```
Comment (marineb) [inline front/style/manager-space/_forms.scss]: Text mixin

File: front/style/manager-space/_tables.scss [hunk 28]
```diff
@@ -0,0 +1,147 @@
...
+                        span { background-color: #2abeaf; }
+                    }
+
+                    &__2 {
+                        color: $silver-grey;
+                        span { border: 2px solid #7b889b; }
+                    }
+
+                    &__3 {
+                        color: #cf4155;
```
Comment (marineb) [inline front/style/manager-space/_tables.scss]: Color variable

File: front/style/manager-space/_tables.scss [hunk 29]
```diff
@@ -0,0 +1,147 @@
...
+                        span { border: 2px solid #cf4155; }
+                    }
+                }
+
+                a {
+                    color: #1d5fd1;
```
Comment (marineb) [inline front/style/manager-space/_tables.scss]: Color variable

File: front/style/manager-space/_tables.scss [hunk 30]
```diff
@@ -0,0 +1,147 @@
...
+                    border-color: #1d5fd1;
```
Comment (marineb) [inline front/style/manager-space/_tables.scss]: Same

File: front/style/manager-space/roles/_referent.scss [hunk 31]
```diff
@@ -1,6 +1,7 @@
 .referent {
+
     h2 {
-        font-family: $font-gillsans;
+        font-family: $font-roboto, $font-gillsans;
```
Comment (marineb) [inline front/style/manager-space/roles/_referent.scss]: Why put the 2 typefaces ? You can remove gillsans if you don't want it in there (roboto already has some fallback fonts).

File: front/style/manager-space/roles/_referent.scss [hunk 32]
```diff
@@ -18,9 +19,9 @@
             font-weight: bold;
         }
 
-        &__form {
-            padding-top: 30px;
-        }
+        // &__form {
```
Comment (marineb) [inline front/style/manager-space/roles/_referent.scss]: Delete that code if unnecessary :)

File: front/style/manager-space/roles/_referent.scss [hunk 33]
```diff
@@ -74,11 +75,15 @@
     }
 
     &__btn-create-event {
```
Comment (marineb) [inline front/style/manager-space/roles/_referent.scss]: Delete this class if it doesn't have anything in it anymore ! 🔥

File: templates/referent/organizational_chart.html.twig [hunk 34]
```diff
@@ -4,7 +4,6 @@
 
 {% block stylesheets %}
     {{ parent() }}
-    <link rel="stylesheet" href="{{ asset('css/organizational_chart.css') }}">
```
Comment (marineb) [inline templates/referent/organizational_chart.html.twig]: Nice !

File: front/style/manager-space/_filters.scss [hunk 35]
```diff
@@ -0,0 +1,182 @@
...
```
Comment (marineb) [inline front/style/manager-space/_filters.scss]: 👆

File: front/style/manager-space/_tables.scss (hunk 28 above)
Comment (marineb) [reply front/style/manager-space/_tables.scss]: 👆

File: front/style/manager-space/roles/_referent.scss (hunk 31 above)
Comment (marineb) [reply front/style/manager-space/roles/_referent.scss]: 👆

**Pull request merged**